### PR TITLE
Ported PAL to tablet

### PR DESCRIPTION
**Pull request by hyperlogic**
The People button on the tablet now launches the PAL UI on the tablet surface itself, instead of the floating HUD.

### CODE REVIEW

**Comment by hyperlogic** [comment]
QA Test:
    * Ensure that all the functionality of the PAL still works.

File: libraries/fbx/src/FBXReader_Node.cpp [hunk 1]
```diff
@@ -62,7 +62,9 @@ template<class T> QVariant readBinaryArray(QDataStream& in, int& position) {
             position += sizeof(T) * arrayLength;
             in.readRawData(arrayData.data(), arrayData.size());
         }
-        memcpy(&values[0], arrayData.constData(), arrayData.size());
+        if (arrayData.size() > 0) {
+            memcpy(&values[0], arrayData.constData(), arrayData.size());
+        }
```
Comment (sethalves) [inline libraries/fbx/src/FBXReader_Node.cpp]: I'm afraid to ask

**Comment by hifi-gustavo** [comment]
Build console output for commit: e25ad0f332837b7594cdd62b3fa1fbf1222c6f26 available at: https://gist.github.com/c87a9e6f4d7f7937d5397423098fa360<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9639-e25ad0f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9639-e25ad0f.dmg

**Comment by nbq** [comment]
Testing this one

**Comment by nbq** [comment]
Tester: Coal
System: i7 6700k, GeForce 970, Windows 10

* Tested PAL in tablet
* Tested Ignore, Mute, Silence and Ban

All worked as they are supposed to

**PASS**

**Pull request merged**